### PR TITLE
Translate reference/config/policy-and-telemetry/metrics into Chinese

### PR DESCRIPTION
**Pull request by xiaosuiba**
[original code](https://github.com/istio/istio.github.io/tree/master/content/docs/reference/config/policy-and-telemetry/metrics.md) [original page](https://preliminary.istio.io/docs/reference/config/policy-and-telemetry/metrics)

### CODE REVIEW

**Comment by rootsongjc** [comment]
@xiaosuiba Strange CI errors again.
```
- ./public/help/troubleshooting/index.html
  *  138:222: ERROR: Unexpected end tag : p (line 138)
htmlproofer 3.9.1 | Error:  HTML-Proofer found 1 failure!
```

**Comment by geeknoid** [comment]
Please rebase and resubmit the PR to get rid of these circle ci errors.

**Comment by rootsongjc** [comment]
@xiaosuiba CI failed again.
```
- ./public/blog/2018/egress-https/index.html
  *  External link https://raw.githubusercontent.com/istio/istio/release-1.0/samples/bookinfo/kube/bookinfo-details-v2.yaml failed: 404 No error
- ./public/blog/2018/egress-tcp/index.html
  *  External link https://raw.githubusercontent.com/istio/istio/release-1.0/samples/bookinfo/kube/bookinfo-ratings-v2-mysql.yaml failed: 404 No error
- ./public/docs/examples/bookinfo/index.html
  *  External link https://raw.githubusercontent.com/istio/istio/release-1.0/samples/bookinfo/consul/bookinfo.yaml failed: 404 No error
  *  External link https://raw.githubusercontent.com/istio/istio/release-1.0/samples/bookinfo/consul/cleanup.sh failed: 404 No error
  *  External link https://raw.githubusercontent.com/istio/istio/release-1.0/samples/bookinfo/kube/bookinfo.yaml failed: 404 No error
  *  External link https://raw.githubusercontent.com/istio/istio/release-1.0/samples/bookinfo/kube/cleanup.sh failed: 404 No error
- ./public/docs/examples/integrating-vms/index.html
  *  External link https://raw.githubusercontent.com/istio/istio/release-1.0/samples/bookinfo/kube/bookinfo-ratings-v2-mysql-vm.yaml failed: 404 No error
- ./public/docs/tasks/security/basic-access-control/index.html
  *  External link https://raw.githubusercontent.com/istio/istio/release-1.0/samples/bookinfo/kube/mixer-rule-deny-label.yaml failed: 404 No error
- ./public/docs/tasks/security/role-based-access-control/index.html
  *  External link https://raw.githubusercontent.com/istio/istio/release-1.0/samples/bookinfo/kube/bookinfo-add-serviceaccount.yaml failed: 404 No error
  *  External link https://raw.githubusercontent.com/istio/istio/release-1.0/samples/bookinfo/kube/istio-rbac-details-reviews.yaml failed: 404 No error
  *  External link https://raw.githubusercontent.com/istio/istio/release-1.0/samples/bookinfo/kube/istio-rbac-enable.yaml failed: 404 No error
  *  External link https://raw.githubusercontent.com/istio/istio/release-1.0/samples/bookinfo/kube/istio-rbac-namespace.yaml failed: 404 No error
  *  External link https://raw.githubusercontent.com/istio/istio/release-1.0/samples/bookinfo/kube/istio-rbac-productpage.yaml failed: 404 No error
  *  External link https://raw.githubusercontent.com/istio/istio/release-1.0/samples/bookinfo/kube/istio-rbac-ratings.yaml failed: 404 No error
- ./public/docs/tasks/telemetry/tcp-metrics/index.html
  *  External link https://raw.githubusercontent.com/istio/istio/release-1.0/samples/bookinfo/kube/bookinfo-db.yaml failed: 404 No error
  *  External link https://raw.githubusercontent.com/istio/istio/release-1.0/samples/bookinfo/kube/bookinfo-ratings-v2.yaml failed: 404 No error
htmlproofer 3.9.1 | Error:  HTML-Proofer found 16 failures!
```

File: content_zh/docs/reference/config/policy-and-telemetry/metrics/index.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+*   **请求大小（Request Size）**：这是一个 `DISTRIBUTION`，它测量 HTTP 请求的 body 大小。
+
+*   **响应大小（Response Size）**：这是一个 `DISTRIBUTION`，它测量 HTTP 响应 body 的大小。
+
+*   **Tcp 发送字节数（Tcp Byte Sent）**：这是一个 `COUNTER`，它测量在 TCP 连接场景下响应期间发送的总字节数，由服务端代理测量。
+
+*   **Tcp 接收字节数（Tcp Byte Received）**：这是一个 `COUNTER`，它测量在 TCP 连接场景下请求期间接收的总字节数，由服务端代理测量。
+
+## 标签（Labels）
```
Comment (rootsongjc) [inline content_zh/docs/reference/config/policy-and-telemetry/metrics/index.md]: 去掉复数

File: content_zh/docs/reference/config/policy-and-telemetry/metrics/index.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
+---
+title: 默认监控指标
+description: 通过 Mixer 从 Istio 导出的默认监控指标（Metrics）。
```
Comment (rootsongjc) [inline content_zh/docs/reference/config/policy-and-telemetry/metrics/index.md]: metrics 去掉复数

File: content_zh/docs/reference/config/policy-and-telemetry/metrics/index.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+    request_protocol: api.protocol | context.protocol | "unknown"
+    {{< /text >}}
+
+*   **响应码（Response Code）**:标识了请求的响应码。该 label 仅在 HTTP metrics 中存在。
+
+    {{< text yaml >}}
+    response_code: response.code | 200
+    {{< /text >}}
+
+*   **连接 mTLS（Connection mTLS）**: 标识请求使用的 service 认证策略。当 Istio 使用身份认证保证通信安全时设置为 `true。
```
Comment (rootsongjc) [inline content_zh/docs/reference/config/policy-and-telemetry/metrics/index.md]: true 后面漏了个符号

File: content_zh/docs/reference/config/policy-and-telemetry/metrics/index.md (hunk 2 above)
Comment (xiaosuiba) [reply content_zh/docs/reference/config/policy-and-telemetry/metrics/index.md]: metrics 这里不是 metric 的复数形式，在软件领域使用 metrics 表示指标或者度量。

Comment (rootsongjc) [reply content_zh/docs/reference/config/policy-and-telemetry/metrics/index.md]: OK

**Pull request merged**